.github/workflows/add-model-like.yml

```diff
@@ -27,14 +27,25 @@ jobs:
         id: cache
         with:
           path: ~/venv/
-          key: v2-tests_model_like-${{ hashFiles('setup.py') }}
+          key: v3-tests_model_like-${{ hashFiles('setup.py') }}
 
       - name: Create virtual environment on cache miss
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
           python -m venv ~/venv && . ~/venv/bin/activate
           pip install --upgrade pip!=21.3
-          pip install .[dev]
+          pip install -e .[dev]
```

Comment:
this is the main change in order to use `/home/runner/work/transformers/transformers/src`

Reply:
This doesn't usually work, as an editable install writes some content in some folders, and this is not allowed on GitHub action `pull-request` (for security reasons, since the user have put code on their own in their PR).

I have no idea why the check passes now, maybe it's not executing this version of the test script.